src/lib.rs

```diff
@@ -75,7 +74,7 @@ struct Bloom {
     /// How many hash functions this filter uses
     n_hash_funcs: u32,
     /// The bit length of the filter
-    size: u32,
+    size: usize,
```

Comment:
One nice thing about keeping this as a `u32` would be that in `next_index` for the SHA256 counter index generator, it would be impossible to ever need more than 4 bytes.

Reply:
Yeah that's probably a good idea. I think I did this because the `bit_len` field of `BitVector` is `usize`.